OxfordBodleian/Juel-Jensen/BDLaethb6.xml

```diff
@@ -50,23 +50,21 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
                             <supportDesc>
                                 <support>
                                     <material key="parchment"/>
-                                    <p></p>
                                 </support>
                                 <extent>
-                                    <measure unit="leaf">188</measure>
+                                    <measure unit="leaf" quantity="188">188</measure>
                                     <dimensions type="outer" unit="mm">
                                         <height>393</height>
                                         <width>343</width>
                                         <depth>90</depth>
                                     </dimensions>
+                                    <note>The manuscript could not be weighed due to its great size.</note>
                                 </extent>
-                                <foliation>The manuscript is foliated in pencil in the upper right
-                                    corner. </foliation>
-                                <!--Pencil foliation in the upper right corner-->
+                                <foliation>Pencil foliation in the upper right corner.</foliation>
                                 <condition key="good">The manuscript is in a good state of preservation.
                                     The binding is slightly damaged, with the leather cover missing in the lower corners of the front board and all corners of the back board.
                                     The spine has been removed, like the additional leather patches on the inner side of the front board, presumably to resew the manuscript.
-                                    The lower margin of the cover is darkened by use. The lower outer corner of the first folia has been damaged probably by rodents.
+                                    The lower margin of the cover is darkened by use. The lower outer corner of the first folios has been damaged probably by rodents.
```

Comment:
Yes, I am going to check the condition part after Thea has merged this branch. I do agree that it is difficult to say if the leather is darkened because of frequent handling.

Reply:
(Sorry if some of my comments came at the wrong time – thank you, Eliana! :) )

Reply:
No worries! :) I take notes!